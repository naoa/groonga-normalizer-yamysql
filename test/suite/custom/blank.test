register normalizers/yamysql

table_create remove_phrases TABLE_PAT_KEY ShortText
load --table remove_phrases
[
{"_key": "の"}
]

normalize NormalizerYaMySQL "クロアチア の戦争" WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL " クロアチアの戦争" WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL "クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL "クロアチアののの戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL " クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL " クロアチア の戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL " クロアチア の 戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL " クロアチア の戦争 " REMOVE_BLANK|WITH_TYPES|WITH_CHECKS
