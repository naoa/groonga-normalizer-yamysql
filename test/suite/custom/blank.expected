register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLSymbolPartofspeechHtml "クロアチア の戦争" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "クロアチア 戦争",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null",
      "kanji",
      "kanji"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      4,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
normalize NormalizerYaMySQLSymbolPartofspeechHtml " クロアチアの戦争" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": " クロアチア 戦争",
    "types": [
      "others",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null",
      "kanji",
      "kanji"
    ],
    "checks": [
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
normalize NormalizerYaMySQLSymbolPartofspeechHtml "クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "クロアチア 戦争 ",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null",
      "kanji",
      "kanji",
      "null"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0,
      1
    ]
  }
]
normalize NormalizerYaMySQLSymbolPartofspeechHtml " クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": " クロアチア 戦争 ",
    "types": [
      "others",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null",
      "kanji",
      "kanji",
      "null"
    ],
    "checks": [
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0,
      1
    ]
  }
]
normalize NormalizerYaMySQLSymbolPartofspeechHtml " クロアチア の戦争 " WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": " クロアチア 戦争 ",
    "types": [
      "others",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null",
      "kanji",
      "kanji",
      "null"
    ],
    "checks": [
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      4,
      3,
      0,
      0,
      3,
      0,
      0,
      1
    ]
  }
]
normalize NormalizerYaMySQLSymbolPartofspeechHtml " クロアチア の戦争 " REMOVE_BLANK|WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "クロアチア 戦争",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "null",
      "kanji",
      "unknown"
    ],
    "checks": [
      4,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      4,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
